Warn and fix missing mime types (#1050)
reactive-python/reactpy: docs/source/about/changelog.rst

Unreleased
----------

**Changed**

> - :pull:`1050` - Warn when operating system has missing mime types, which can result in ``reactpy.run`` not working as expected.

Reviewer: This will also warn and fix the issue if you're running in production.

```suggestion
- :pull:`1050` - Warn, and attempt to fix, when operating system has missing mime types.
```
